Dockerfile

```diff
@@ -0,0 +1,14 @@
+# USAGE EXAMPLES
+# docker build -t insider .
+# docker run -ti --rm insider -tech android -target $pwd
+# docker run -ti --rm insider -tech javascript -target <myprojectfolder>
```

Comment:
To use this examples we need to map volume of path of project to scan into container.